api/types.go

```diff
@@ -141,27 +142,55 @@ type SeriesType string
 // end =   0 mod resolution
 //
 // This range is inclusive of Start and End (i.e. [Start, End]). Start and End
-// are Unix second timestamps. Resolution is in seconds.
-// TODO: Make these ms
+// are Unix milliseconds timestamps. Resolution is in milliseconds.
+// (Millisecond precision allows an effective range of 290 million years in each direction)
```

Comment:
Probably unnecessary to note that millis range gives us space to last until our robot overloads take over and life ends :P